exonum/src/node/mod.rs

```diff
@@ -328,6 +330,46 @@ impl NodeRole {
     }
 }
 
+#[derive(Debug, Default, Clone, Serialize, Deserialize)]
+/// ConnectList representation in node's config file.
+pub struct ConnectListConfig {
+    /// Peers to which we can connect.
+    pub peers: Vec<ConnectInfo>,
+}
+
+impl ConnectListConfig {
+    /// Creates `ConnectList` from validators public configs.
+    pub fn from_node_config(list: &[NodePublicConfig]) -> Self {
+        let peers: Vec<_> = list.iter()
```

Comment:
Is this `: Vec<_>` really required? Rust should be able to infer a type from its direct use as a Vec field in the next statement.

Reply:
Type isn't needed here, right?